src/index.html

```diff
@@ -3,12 +3,35 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <main>
+    <header class="header-1">
+      <a href="#home" class="logo" >
+        <img
+        src="images/logo.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <a class="link is-active" href="apple">apple</a>
+          <a class="link" href="samsung">samsung</a>
```

Comment:
`ul` should contain only `li` put link into `li`. Also try use BEM naming
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="apple">apple</a>
          </li>
        .... and so on
```